packages/backend/src/notifications.ts

```diff
@@ -4,9 +4,17 @@ import { normalizeEnsName } from '@dm3-org/dm3-lib-profile';
 import express from 'express';
 import { auth } from './utils';
 import { validateNotificationChannel } from './validation/notification/notificationChannelValidation';
+import {
+    ChannelNotSupportedError,
+    DeliveryServiceProperties,
+    addNewNotificationChannel,
+} from '@dm3-org/dm3-lib-delivery';
+import { IDatabase } from './persistance/getDatabase';
 
 // Exporting a function that returns an Express router
-export default () => {
+export default (
```

Comment:
why not passing the DS properties object directly ?

Reply:
I thought if needed to pass path to config file then function would be useful.